stacker/blueprints/empire/empire_controller.py

```diff
@@ -0,0 +1,305 @@
+from troposphere import Ref, Join, Output, GetAtt, Not, Equals, If, FindInMap
+from troposphere import ec2, autoscaling
+from troposphere.autoscaling import Tag as ASTag
+from troposphere.iam import InstanceProfile, Policy, Role
+from troposphere import elasticloadbalancing as elb
+from troposphere.route53 import RecordSetType
+
+from awacs.helpers.trust import (
+    get_default_assumerole_policy, get_ecs_assumerole_policy
+)
+
+from .empire_base import EmpireBase
+
+from .policies import (
+    service_role_policy,
+    empire_policy
+)
+
+CLUSTER_SG_NAME = "EmpireControllerSecurityGroup"
+ELB_SG_NAME = "EmpireControllerELBSecurityGroup"
+
+
+class EmpireController(EmpireBase):
+    PARAMETERS = {
+        'VpcId': {'type': 'AWS::EC2::VPC::Id', 'description': 'Vpc Id'},
+        'DefaultSG': {'type': 'AWS::EC2::SecurityGroup::Id',
+                      'description': 'Top level security group.'},
+        'ExternalDomain': {
+            'type': 'String',
+            'description': 'Base domain for the stack.',
+            'default': ''},
+        "InternalZoneId": {
+            "type": "AWS::Route53::HostedZone::Id",
+            "description": "Zone ID of the internal Empire zone",
+            },
+        'PrivateSubnets': {'type': 'List<AWS::EC2::Subnet::Id>',
+                           'description': 'Subnets to deploy private '
+                                          'instances in.'},
+        'PublicSubnets': {'type': 'List<AWS::EC2::Subnet::Id>',
+                          'description': 'Subnets to deploy public (elb) '
+                                         'instances in.'},
+        'AvailabilityZones': {'type': 'CommaDelimitedList',
+                              'description': 'Availability Zones to deploy '
+                                             'instances in.'},
+        'InstanceType': {'type': 'String',
+                         'description': 'Empire AWS Instance Type',
+                         'default': 'm3.medium'},
+        "MinHosts": {"type": "Number",
+                     "description": "Minimum # of empire minion instances.",
+                     "default": "2"},
+        "MaxHosts": {"type": "Number",
```

Comment:
Super minor, but this formatting is a little hard to read. Is there some pep recommended format? I think it would be easier to read as (NL after key):

```python
"MaxHosts": {
  "type": "Number",
  "description": "",
}
```

Also, mixed double and single quotes.